gcloud/connection.py

```diff
@@ -219,6 +225,10 @@ def _do_request(self, method, url, headers, data):
         :type data: string
         :param data: The data to send as the body of the request.
 
+        :type dummy: object or :class:`NoneType`
+        :param dummy: Unused ``target_object`` here but may be used
+                      by a superclass.
```

Comment:
I think the superclass argument names should match those we expect to be used in the subclasses (the name is part of the contract).  Because such uses are common, pylint ought not to bitch about an unused argument (rather than an explicit local).